core/src/main/java/org/opensearch/sql/catalog/CatalogService.java

```diff
@@ -0,0 +1,20 @@
+package org.opensearch.sql.catalog;
```

Comment:
license header

Reply:
Ok is there any plugin for this to cause build failures in case of missing headers.

Reply:
yes `./gradlew opensearch-sql-plugin:licenseHeaders` from opensearch, not sure why it doesn't fail
https://github.com/opensearch-project/sql/blob/d56f442ab573cc13f4cf85f04fc2bce0f289eefc/plugin/build.gradle#L126-L133